qiita_db/handlers/artifact.py

```diff
@@ -69,6 +69,46 @@ def get(self, artifact_id):
 
         self.write(response)
 
+    @authenticate_oauth
+    def patch(self, artifact_id):
+        """Patches the filepaths of the artifact
+
+        Parameter
+        ---------
+        artifact_id : str
+            The id of the artifact whole filepaths information is being updated
+
+        Returns
+        -------
+        dict
+            Format:
+            {'success': bool,
```

Comment:
On previous return dictionaries, we were using a slightly different scheme, is there a good motivation to change that? I'm referring to the dictionaries that have a "status" and a "message" key, etc.